src/Main/Main.js

```diff
@@ -0,0 +1,25 @@
+import './Main.scss'
+import burgerImage from './burgers.webp'
+import Navbar from '../Navbar/Navbar'
+import BannerBoxFoodDelivered from '../BannerBoxFoodDelivered/BannerBoxFoodDelivered'
+
+function Main() {
+  return (
+    <main className="Main">
```

Comment:
the classname is created but not used in the styling file, so I wonder why you have used it?

Reply:
agreed